lock-api/src/main/java/com/palantir/lock/client/NamespacedConjureTimelockService.java

```diff
@@ -24,54 +24,27 @@
 import com.palantir.atlasdb.timelock.api.ConjureRefreshLocksResponse;
 import com.palantir.atlasdb.timelock.api.ConjureStartTransactionsRequest;
 import com.palantir.atlasdb.timelock.api.ConjureStartTransactionsResponse;
-import com.palantir.atlasdb.timelock.api.ConjureTimelockService;
 import com.palantir.atlasdb.timelock.api.ConjureUnlockRequest;
 import com.palantir.atlasdb.timelock.api.ConjureUnlockResponse;
 import com.palantir.atlasdb.timelock.api.ConjureWaitForLocksResponse;
 import com.palantir.atlasdb.timelock.api.GetCommitTimestampsRequest;
 import com.palantir.atlasdb.timelock.api.GetCommitTimestampsResponse;
 import com.palantir.lock.v2.LeaderTime;
-import com.palantir.tokens.auth.AuthHeader;
 
-public class NamespacedConjureTimelockService {
-    private static final AuthHeader AUTH_HEADER = AuthHeader.valueOf("Bearer omitted");
-    private final String namespace;
-    private final ConjureTimelockService conjureTimelockService;
+public interface NamespacedConjureTimelockService {
+    ConjureUnlockResponse unlock(ConjureUnlockRequest request);
 
-    public NamespacedConjureTimelockService(ConjureTimelockService conjureTimelockService, String namespace) {
```

Comment:
This will have a number of downstream breaks in the internal backup product and internal timelock migration CLI tools. Overall impact is limited, but we need to ensure these get fixed.